oai_agents/agents/rl.py

```diff
@@ -260,9 +265,25 @@ def should_evaluate(self, steps):
 
         steps_divisable_by_5 = (steps + 1) % 5 == 0
         mean_rew_greater_than_best = mean_training_rew > self.best_training_rew and self.learning_agent.num_timesteps >= 5e6
-        fcp_ck_rate_reached = self.fcp_ck_rate and self.learning_agent.num_timesteps // self.fcp_ck_rate > (len(self.ck_list) - 1)
+        checkpoint_rate_reached = self.checkpoint_rate and self.learning_agent.num_timesteps // self.checkpoint_rate > (len(self.ck_list) - 1)
     
-        return steps_divisable_by_5 or mean_rew_greater_than_best or fcp_ck_rate_reached
+        return steps_divisable_by_5 or mean_rew_greater_than_best or checkpoint_rate_reached
+
+    def log_details(self, experiment_name, total_train_timesteps):
+        print("Training agent: " + self.name + ", for experiment: " + experiment_name)
+        self.print_tc_helper(self.teammates_collection, "Train TC")
+        self.print_tc_helper(self.eval_teammates_collection, "Eval TC")
+        self.curriculum.print_curriculum()
+        print("How Long: ", self.args.how_long)
+        print(f"Epoch timesteps: {self.epoch_timesteps}")
+        print(f"Total training timesteps: {total_train_timesteps}")
+        print(f"Number of environments: {self.n_envs}")
+        print(f"Hidden dimension: {self.hidden_dim}")
+        print(f"Seed: {self.seed}")
+        print(f"Checkpoint rate: {self.checkpoint_rate}")
+        print(f"Learner type: {self.learner_type}")
+        print("Dynamic Reward? ", self.args.dynamic_reward)
```

Comment:
is the question mark"?" a typo in print("Dynamic Reward? ", self.args.dynamic_reward)?

suggested edits:
`print("Dynamic Reward: ", self.args.dynamic_reward)`